recipe/meta.yaml

```diff
@@ -57,10 +56,7 @@ outputs:
     # When building out the initial package set for a new Python version the
     # recommendataion is to build numpy-base but not numpy, then build
     # mkl_fft and mkl_random, and then numpy.
-    # The false line will build numpy-base only
-    # {% if false %}
-    # The true line will build numpy-base and numpy
-    {% if true %}
+    {% if only_build_numpy_base != 'yes' %}
```

Comment:
bit confusing, `== 'no'` is slightly easier, or even better something like `if build_full == 'yes'`